### ISSUE TITLE
Icon for collapsed event subprocess

### ISSUE THREAD

**Issue by rajgoel** [bpmn-io/bpmn-font#24]
### Is your feature request related to a problem? Please describe.

Currently, there is only an icon for expanded event subprocesses, but not for collapsed ones. 

### Describe the solution you'd like

It would be nice to have an icon for collapsed event subprocesses, i.e. analogously to the icon for collapsed subprocesses (with the + marker).

### Describe alternatives you've considered

Continue using the icon for expanded event subprocesses, but this is inconsistent with the icons used in the replace popup menu for other elements.

### Additional context

This repo provides a module to collapse/expand event subprocesses from the replace popup menu: https://github.com/bpmn-os/bpmn-js-collapse-event-subprocess/

**Comment by nikku** [comment]
Makes sense. We're happy to incorporate one if you contribute it. Moving to backlog.
